### DISCUSSION TITLE
xcodebuild[73837:414611] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition

### DISCUSSION THREAD

**Discussion by ChristineWasike** [fastlane/fastlane#20921] [General]
I recently ran into this error when running `fastlane beta` to upload my app to testflight.
I shared more details on this in a recent issue [#20919](https://github.com/fastlane/fastlane/issues/20919) I just opened.
I hope to learn from the community and get this resolved soon. Thanks!

**Comment by jaaywags** [comment]
@ChristineWasike did you ever resolve this? 🤞

**Reply by jaaywags** [reply]
nvm, I got passed this by upgrading React Native from 0.68.x to 0.71.2, and changing XCode version from 14.0.1 to 13.2.1.

**Comment by shamnex** [comment]
Having the same issue. How were you able to solve it ? @ChristineWasike

**Reply by ChristineWasike** [reply]
Try checking out this issue: https://github.com/fastlane/fastlane/issues/21071

**Reply by maskowe** [reply]
> Try checking out this issue: #21071

@ChristineWasike  I'm getting the fastlane error, but I don't have any issue with my Apple account or password.   I'm on the latest eas cli also - do you have any idea what is causing this ios preview build error for me?  Thanks.

**Comment by markst** [comment]
Also been experiencing this:
```
2023-05-24 13:21:28.014 xcodebuild[80420:3303409] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition
```

**Reply by titanium-cranium** [reply]
@markst Any insight on where this came from and how it was resolved?

**Comment by 0xtruly** [comment]
I am currently facing the same issue

**Reply by bytenaija** [reply]
Yeah. I am experiencing this issue on Xcode 15.1

`2024-01-01 17:18:35.280 xcodebuild[9234:82904] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition`

**Reply by easydev991** [reply]
I have the same issue. Does anyone have a solution?

**Comment by rricamar** [comment]
For any wondering how to solve this error, the solution was posted on the issue mentioned in the first comment (https://github.com/fastlane/fastlane/issues/20919)

In my case, I just put `ENV["FASTLANE_XCODEBUILD_SETTINGS_RETRIES"] = "9"` on the very top of my `Fastfile`. Since each retry will increment the `timeout`, it will eventually work 👍
